### DISCUSSION TITLE
Transforms: Concatenation of fields

### DISCUSSION THREAD

**Discussion by maemigh** [grafana/grafana#38375] [Dashboards, Panels & Visualizations]
The new field from calculation is a great feature. It would be nice if fields could also be concatenated.

**What would you like to be added**:
Calculated fields by string concatenation

**Why is this needed**:
Can simplify queries or provide concat functionality where datasources do not

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by shashwat-kalpvaig** [comment]
I wanted string concat for feild value for basic addition of root url for building the url. It seems pretty simple but i cannot find the feature in grafana transformation query.

**Comment by vaibhav-pan** [comment]
This feature is still not available?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424).

Looking at this request, I think it is possible with existing transformations. If you are getting values added instead of concatinated, it might be a matter of converting the field type to be a string instead of a number. But after that, doing a binary operation with a string will concatinate the values together. 
<img width="719" alt="Screenshot 2024-02-17 at 1 11 38 PM" src="https://github.com/grafana/grafana/assets/1533128/e0970685-c4d2-4826-8dfc-7d3942fac543">

Feel free to make an [issue](https://github.com/grafana/grafana/issues) if there is something I am missing from this request and you would like to request the feature. Thank you!

**Reply by yerycs** [reply]
There can be cases some deliimiter is needed, for example `a/0`.
`/` is needed to concat two fields, but it is impossible to implement in transformation
